### PR TITLE
[YUNIKORN-56] Broken Scheduler configuration link in deployment.md

### PR DESCRIPTION
**Pull request by kingamarton**

### CODE REVIEW

**Comment by kingamarton** [comment]
The change touches only the documentation. It should not cause unit test failures. @yangwwei can you retrigger the check?

**Comment by yangwwei** [comment]
Hi @kingamarton 

You can click the `detail` or the `checks`, expand the check and there is a rerun button on the right side corner. No worries, the failure should not be related. We've found some flaky tests on github, we will need to fix them to get a more stable action execution.

File: docs/setup/deployment.md [hunk 1]
```diff
@@ -56,4 +56,4 @@ kubectl port-forward ${POD} 9889 9080
 ```
 
 `9889` is the default port for Web UI, `9080` is the default port of scheduler's Restful service where web UI retrieves info from.
-Once this is done, web UI will be available at: http://localhost:9889.
\ No newline at end of file
+Once this is done, web UI will be available at: http://localhost:9889.
```
Comment (yangwwei) [inline docs/setup/deployment.md]: what has changed on this line?

Comment (mKinga) [reply docs/setup/deployment.md]: This was unintentional. I checked the two files and I introduces a newline by mistake. I will remove it.

Comment (kingamarton) [reply docs/setup/deployment.md]: This was unintentional. I checked the two files and I introduces a newline by mistake. I will remove it.

**Comment by yangwwei** [comment]
I guess this is good. +1

**Pull request merged**